build/ol/install-packages.sh

```diff
@@ -33,6 +33,6 @@ dnf install -y ImageMagick \
   unzip \
   zstd \
   lbzip2 \
-  knock
+  libpcap0.8
```

Comment:
Were you able to confirm this is the package name for DNF?

If not yet, you can by using a build arg as:

```
docker build --build-arg BASE_IMAGE=ghcr.io/graalvm/graalvm-ce:ol8-java17 .
```

Reply:
Thanks, it was wrong in fact